### PR TITLE
Adjust AmgX to use JACOBI_L1

### PR DESCRIPTION
**Pull request by artv3**
This PR also fixes a bug when using AmgX defaults as preconditioner - I was hitting errors based on how it was configured. 

 | PR | Author | Editor | Reviewers | Assignment | Approval | Merge| 
 | --- | --- | --- | --- | --- | --- | --- | 
| [#2345](https://github.com/mfem/mfem/pull/2345) | @artv3 | @tzanio | @pazner + @dylan-copeland | 06/22/21 | 06/29/21 | 06/30/21 | |

### CODE REVIEW

**Comment by tzanio** [comment]
This PR is now [under review](https://github.com/mfem/mfem/blob/master/CONTRIBUTING.md#pull-requests) (see the table in the PR description). To help with the review process, please **do not force push** to the branch.

**Comment by dylan-copeland** [comment]
Note that PR #2348 is necessary for testing with the latest version of hypre. In the meantime, for testing PRs like this, we have to use an older version of hypre.

File: linalg/amgxsolver.cpp [hunk 1]
```diff
@@ -208,11 +208,14 @@ void AmgXSolver::DefaultParameters(const AMGX_MODE amgxMode_,
                     " \"config_version\": 2, \n"
                     " \"solver\": { \n"
                     "   \"solver\": \"AMG\", \n"
+                    "  \"scope\": \"main\", \n"
```
Comment (dylan-copeland) [inline linalg/amgxsolver.cpp:211]: This could be better aligned with one more space.

**Review by dylan-copeland — APPROVED** [review]
Thanks @artv3! I ran all the sample runs on lassen in examples/amgx, and everything worked fine.

**Review by tzanio — APPROVED** [review]

**Comment by tzanio** [comment]
Merged in `next` for testing...

**Pull request merged**